mattermost-plugin/webapp/src/error_boundary.tsx

```diff
@@ -2,27 +2,43 @@
 // See LICENSE.txt for license information.
 
 import React from 'react'
+import PropTypes from 'prop-types'
+
+import {Utils} from '../../../webapp/src/utils'
 
-type Props = {}
 type State = {
     hasError: boolean
 }
 
-export default class ErrorBoundary<Props, State> extends React.Component {
+export default class ErrorBoundary extends React.Component {
     state = {hasError: false}
+    propTypes = {children: PropTypes.node.isRequired}
+    msg = 'Redirecting to error page...'
```

Comment:
Should this be translated?

Reply:
I'll add it to translation just in case, but nobody should ever see that since the error boundary always redirects to the error page.

Reply:
@sbishel considering this is in the plugin webapp, and it currently has not i18n strings, I think we should leave this untranslated.  Otherwise we have create a new set of translation files to go here https://github.com/mattermost/focalboard/tree/main/mattermost-plugin/webapp/i18n